metadata-ingestion/src/datahub/ingestion/source/dbt/dbt_common.py

```diff
@@ -1352,7 +1352,9 @@ def _create_subType_wu(
         subtypes: List[str] = [node.node_type.capitalize()]
         if node.materialization == "table":
```

Comment:
minor nit: can these typenames go into an enum?

Reply:
Ideally yes, but that's a bigger refactor - don't want to do that now